spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/PubSubEmulator.java

```diff
@@ -59,29 +62,34 @@
 	// Hostname for cleanup, should always be localhost.
 	private String emulatorHostPort;
 
-	// Allow gating rule execution on a flag.
-	private boolean ruleDisabled;
+	// Conditional rule execution based on an environmental flag.
+	private boolean enableTests;
 
 	public PubSubEmulator() {
-		if (!"true".equals(System.getProperty("it.emulator"))) {
-			LOGGER.warn("PubSubEmulator rule disabled. Please enable with -Dit.emulator");
-			this.ruleDisabled = true;
+		if ("true".equals(System.getProperty("it.pubsub-emulator"))) {
+			this.enableTests = true;
+		}
+		else {
+			LOGGER.warn("PubSubEmulator rule disabled. Please enable with -Dit.pubsub-emulator");
```

Comment:
`-Dit.pubsub-emulator=true` right?

Reply:
Maven will default properties to "true" by default, so either way, I think.
https://github.com/apache/maven/blob/864d180edf7b66b07e2e3fc08709b5cb1e4f051c/maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java#L1690

Reply:
Ok, cool!